infra/examples-dev/gcp/google-workspace-variables.tf

```diff
@@ -26,6 +26,12 @@ variable "google_workspace_example_admin" {
   default     = null # will failover to user
 }
 
+variable "google_workspace_provision_keys" {
```

Comment:
Maybe the name could be `provision_google_workspace_keys` to clarify this is the flag for creating the keys, not the *keys* itself to use.

Reply:
yeah ... i was just trying group all the variables with common prefix, to make them easy to find by source at this level.